### PR TITLE
Accenture CTI Logo & bug fix

### PR DESCRIPTION
**Pull request by satyakidroid**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
1. Had minor issues with showing up `LastSeen` field. It popped up error for those network indicators where `LastSeen` was'nt available in `IntelGraph` platform. Have fixed that issue✅
2. Had issues with logo and would be fixed with this PR✅
fixes: link to the issue

## Description
Just to get bugs fixed 

## Screenshots
Those having `LastSeen` field:
image

Those not having `LastSeen` field:
image

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/iDefense/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Accenture CTI",
     "description": "Accenture Security",
     "support": "partner",
-    "currentVersion": "3.2.2",
+    "currentVersion": "3.3.0",
```
Comment (merit-maita) [inline Packs/iDefense/pack_metadata.json:5]: its a revision change does not have to be minor
```suggestion
    "currentVersion": "3.2.3",
```

**Review by merit-maita — CHANGES_REQUESTED** [review]
thanks!

File: Packs/AccentureCTI/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Accenture CTI v2",
     "description": "Accenture Cyber Threat Intelligence",
     "support": "partner",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.1.0",
```
[the remark this answers is not in the document]
Comment (satyakidroid) [reply Packs/AccentureCTI/pack_metadata.json]: Done

File: Packs/iDefense/pack_metadata.json (hunk 1 above)
Comment (satyakidroid) [reply Packs/iDefense/pack_metadata.json]: Done

**Comment by satyakidroid** [comment]
For this, should we add links to our new Packs `AccentureCTI` & `AccentureCTI_Feed` , so that users can migrate to?
https://github.com/iDefense/content/blob/2f0c4b7d3a01069618fb79c075da93bbb16d2dd8/Packs/iDefense/README.md?plain=1#L1

**Comment by kgal-pan** [comment]
> For this, should we add links to our new Packs `AccentureCTI` & `AccentureCTI_Feed` , so that users can migrate to? [iDefense/content@`2f0c4b7`/Packs/iDefense/README.md?plain=1#L1](https://github.com/iDefense/content/blob/2f0c4b7d3a01069618fb79c075da93bbb16d2dd8/Packs/iDefense/README.md?plain=1#L1)

@satyakidroid - Yes, this a good idea!

**Review by merit-maita — APPROVED** [review]

**Pull request merged**